internal/queryexecutor/range_query_planner.go

```diff
@@ -1,28 +1,43 @@
 package queryexecutor
 
 import (
-	"errors"
-
 	"github.com/IBM-Blockchain/bcdb-server/internal/stateindex"
 	"github.com/IBM-Blockchain/bcdb-server/pkg/constants"
 )
 
 type rangeQueryPlan struct {
 	startKey *stateindex.IndexEntry
 	endKey   *stateindex.IndexEntry
+	seekKey  *stateindex.IndexEntry
```

Comment:
I would call that `excludeKeys` and make it an array, to allow multiple $neq conditions in a range